shotover-proxy/src/transforms/redis/cache.rs

```diff
@@ -1,24 +1,24 @@
+use std::borrow::Borrow;
 use std::collections::HashMap;
 
-use anyhow::{anyhow, Result};
+use anyhow::{anyhow, bail, Result};
 use async_trait::async_trait;
+use bytes::{BufMut, Bytes, BytesMut};
+use cassandra_protocol::frame::Version;
+use itertools::Itertools;
 use serde::Deserialize;
+use sqlparser::ast::{Assignment, BinaryOperator, Expr, Ident, Query, SetExpr, Statement, Value};
 use tracing::info;
 
```

Comment:
Oh I never noticed that rust fmt put the crate imports at the top!
Your approach is way easier then, lets do that.

But lets also always prefix our internal imports with `crate::` (instead of say `use child_mod::Foo`) so that our internal imports are at the top.